src/bio/bio_recovery.c

```diff
@@ -66,7 +66,7 @@ on_faulty(struct bio_blobstore *bbs)
 
 	rc = ract_ops->faulty_reaction(tgt_ids, tgt_cnt);
 	if (rc < 0)
-		D_ERROR("Faulty reaction failed. %d\n", rc);
+		D_ERROR("Faulty reaction failed. "DF_RC"\n", DP_RC(rc));
```

Comment:
these ones probably aren't either.

Reply:
This one is DER.
.faulty_reaction        = nvme_faulty_reaction,
nvme_faulty_reaction returns -DER